tests/conftest.py

```diff
@@ -151,3 +154,59 @@ def run(self, event):
 
     app.extensions['invenio-webhooks'].register('test-receiver', TestReceiver)
     return TestReceiver
+
+
+@pytest.fixture
+def task_receiver(app):
+    """Register test celery task receiver."""
+
+    @shared_task
+    def add(x, y):
+        TestCeleryTaskReceiver.result = x + y
```

Comment:
It's just an ad-hoc hack in order to get the result to see every task executed properly. The real issue with testing it is that I cannot make any status updates inside tasks and monitor them from the test (`ALWAYS_EAGER=True`).